chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/listing/reviewer/TestFunctionalityReviewer.java

```diff
@@ -22,12 +31,31 @@
  *
  */
 @Component("testFunctionalityReviewer")
-public class TestFunctionalityReviewer implements Reviewer {
+@Transactional
+public class TestFunctionalityReviewer implements Reviewer, ApplicationListener<ContextRefreshedEvent> {
     private static final String EDITION_2014 = "2014";
+    private static final String EDITION_2015 = "2015";
```

Comment:
I think there is a package for 2014-only reviewers and another package for 2015-only reviewers. Could you break up this class to have 2014 logic in the one and 2015 logic the other? Would also require a change to the validators to add the appropriate reviewer.